core/src/snowcli/plugin/load_external_plugins.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import List
+
+import pluggy
+
+from snowcli.cli.common.snow_cli_global_context import global_context_copy
+from snowcli.config import cli_config
+from snowcli.plugin import (
+    ExternalPluginsLoadingMode,
+    ExternalLoadedPlugin,
+    ExternalPluginPackageInfo,
+)
+from snowcli.plugin.api import plugin_hook_specs
+
+log = logging.getLogger(__name__)
+
+
+class LoadExternalPlugins:
```

Comment:
Comment from Barry:

We should think about possibility to configure the path where plugins will be loaded from to allow for installation of them in a specific location.